[CONTINT-4938] Control plane monitoring with Helm on EKS/OpenShift (#33166)
DataDog/documentation: content/en/containers/kubernetes/control_plane.md

## Kubernetes on OpenShift 4 {#OpenShift4}

<div class="alert alert-info">This feature is in Preview.</div>

Datadog supports monitoring Kubernetes Control Plane components, including the API Server, etcd, Controller Manager, and Scheduler.

{{< tabs >}}

> {{% tab "Datadog Operator" %}}

Reviewer: Just noting there are a couple of broken link on this tab:

1. `[datadog-operator Helm chart][12]`. 
2. `Or, for OpenShift users who installed the operator through OperatorHub/Marketplace (the recommended method)`

For links that are on a tab, the link reference must be contained within the tab. For example, for `[datadog-operator Helm chart][12]`, you would put `[12]: https://github.com/DataDog/helm-charts/tree/main/charts/datadog-operator` before the closing `{{% /tab %}}` element.

  oc apply -f -
```

{{% /tab %}}
{{% tab "Helm" %}}

#### Prerequisites

1. Helm chart version >= `3.150.0`
1. Datadog Agent >= `v7.69`

**Note**: `etcd` is not supported on versions 4.0-4.13.

#### General setup

Enable control plane monitoring using the `providers.openshift.controlPlaneMonitoring` option:

{{< code-block lang="yaml" filename="datadog-values.yaml" >}}
datadog:
  apiKey: <DATADOG_API_KEY>
  appKey: <DATADOG_APP_KEY>
  clusterName: <CLUSTER_NAME>
providers:
  openshift:
    controlPlaneMonitoring: true
{{< /code-block >}}

> For OpenShift 4.14 and higher, etcd monitoring requires copying certificates. To copy them into the same namespace the Datadog Agent is running in:

Reviewer: ```suggestion
For OpenShift 4.14 and later, etcd monitoring requires you to copy the etcd certificates. To copy them into the same namespace as the Datadog Agent:
```
